File: src/main/java/com/leona/guess/game.java
package com.leona.guess;

import javax.jws.soap.SOAPBinding;
import java.util.Random;
import java.util.Scanner;

public class game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10)+1; //1-10
        System.out.println(secret); //正確數字
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret){
            System.out.print("Please input a number : ");
            number = scanner.nextInt();
            if (number < secret){
                System.out.println("Higher");
            } else if (number > secret){
                System.out.println("Lower");
            }else{
                System.out.println("Bingo , The number is " + number);
            }
        }

    }
}

File: src/main/kotlin/com/leona/game.kt
package com.leona

import java.util.*
import kotlin.random.Random

fun main(args: Array<String>) {
    val secret = Random.nextInt(10) + 1
    println(secret)
    var number = 0
    while ( number != secret) {
        val scanner = Scanner(System.`in`)
        print("Please input a number :")
        number = scanner.nextInt()
        if (number > secret){
            println("Lower")
        }else if (number < secret) {
            println("Higher")
        }else {
            println("Bingo , the number is $number")
        }
    }
}